pcsx2/CDVD/CsoFileReader.cpp

```diff
@@ -23,6 +23,7 @@
 #include "common/StringUtil.h"
 
 #include <zlib.h>
+#include <lz4.h>
```

Comment:
```suggestion
#include "lz4.h"
```

Since PCSX2 follows the pattern of "system includes use angle brackets, local/thirdparty includes use quotes".

Reply:
But then why is zlib.h just above using angle brackets? They both should use the same style, whichever it is, but I find it weird to have zlib.h use one style and lz4.h use another.

Reply:
zlib is a system header on Linux/Mac.

Reply:
Ok that kinda makes sense, it's still weird, but makes sense.